packages/modals/src/elements/TooltipModal/TooltipModal.tsx

```diff
@@ -86,7 +86,7 @@ const TooltipModalComponent = React.forwardRef<HTMLDivElement, ITooltipModalProp
 
     // Derive aria attributes from props
     const attribute = hasTitle ? 'aria-labelledby' : 'aria-label';
-    const defaultValue = hasTitle ? props['aria-labelledby'] : 'Modal dialog';
+    const defaultValue = hasTitle ? modalProps['aria-labelledby'] : 'Modal dialog';
```

Comment:
Should we differentiate by defaulting to "Tooltip modal dialog" here?

Reply:
I debated that as well and I'm not sure. The components will functionally be treated as a modal regardless, but not all SR users are fully blind and may appreciate the extra identification. I'll check in with a couple folks and get some opinions. 🙂

Reply:
Talking with Jen, we agreed that the concepts of "Drawer modal" and "Tooltip modal" likely wouldn't have a ton of value beyond simply using "modal dialog." What do you think? I could go either way, but leaning towards the generic label.

Reply:
I can appreciate that perspective. Let's remain generic.